### PR TITLE
Strip colons from statsD tags

### PR DESCRIPTION
**Pull request by annaswims**
## Description of change
statsd tags use ':' as a delimiter, so strip them from the values.
## Original issue(s)
https://github.com/department-of-veterans-affairs/va.gov-team/issues/13265

### CODE REVIEW

File: lib/common/client/concerns/monitoring.rb [hunk 1]
```diff
@@ -23,7 +23,8 @@ def increment_total(caller)
         end
 
         def increment_failure(caller, error)
```
Comment (erluti) [inline lib/common/client/concerns/monitoring.rb:25]: any chance you could introduce a test for `increment_failure`?

**Review by erluti — COMMENTED** [review]
LGTM, would love if you introduced a test though

File: lib/common/client/concerns/monitoring.rb (hunk 1 above)
Comment (annaswims) [reply lib/common/client/concerns/monitoring.rb:25]: yep! I added a test.

Comment (erluti) [reply lib/common/client/concerns/monitoring.rb:25]: I'm not seeing any changes to `common/client/concerns/monitoring_spec.rb`, did you push it?

Comment (annaswims) [reply lib/common/client/concerns/monitoring.rb:25]: oopsie! Now I pushed the commit. Thanks!

**Review by erluti — APPROVED** [review]
Thanks for adding a test!

**Pull request merged**